source/catalog_service_API.rst

```diff
@@ -0,0 +1,114 @@
+Catalog Service API
+""""""""""""""""""""
+
+The software library used to implement the Catalogue Service API is based on the `PyCSW <https://pycsw.org/>`__  python
+library.  PyCSW enables users to publish a catalogue of dataset descriptions (metadata) to the web,
```

Comment:
From this point until "Notes for S-ENDA development", this document is a mere copy and paste from this page: https://osgeo.github.io/OSGeoLive-doc/en/overview/pycsw_overview.html

It is enough to add a link. I will never accept copy-and-paste. If the source changes, our text is immediately out of date, so we need to stick to the DRY principle (Don't Repeat Yourself [or others]). In addition, you have issues regarding intellectual property. As it is written now, it appears like we/you are the author(s) of that text. This is clearly wrong.

What we need in the S-ENDA docs is a recipe for how to use pyCSW in our context. 

Please update, and describe how to use pyCSW with our PoC prototype.

Reply:
this should be fixed by latest commit https://github.com/metno/S-ENDA-documentation/pull/68/commits/47c523199dae6c61bd0500a9037ee8d700066cbc